jkamz/questioner-frontend
# 164131692 fetch all question comments

### What does this PR do?
This PR consumes the API for getting all comments for a question

### Description of the tasks to be completed
Consume the API for getting all comment records for a given question
Consume the API for getting details about a specific question

### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/questioner-frontend`

`$ cd questioner-frontend`

Check out to the PR's branch
`$ git checkout ft_fetch_comments_164131692`

### What are the relevant Pivotal Tracker stories?
#### 164131692
#### https://www.pivotaltracker.com/story/show/164131692

### Screeenshots

![getallcomms](https://user-images.githubusercontent.com/11965186/53407928-5d20fd00-39ce-11e9-8fb5-ab6ac9f78eb2.png)
